src/Replace/ClassmapReplacer.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace CoenJacobs\Mozart\Replace;
+
+class ClassmapReplacer extends BaseReplacer
+{
+    /** @var array */
+    public $replacedClasses = [];
+
+    public function replace( $contents )
+    {
+        return preg_replace_callback(
+            '/(?:\W[abstract]*class |interface )([a-zA-Z\_]+)[ ]*{/U',
+            function ($matches) {
+                $replace = 'CJDT_' . $matches[1];
```

Comment:
The `CJDT_` prefix is hardcoded here, needs to be replaced by a config variable.